extensions/amp-facebook-like/0.1/validator-amp-facebook-like.protoascii

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-facebook-like
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-facebook-like extension .js script"
+  requires: "amp-facebook-like"
+  extension_spec: {
+    name: "amp-facebook-like"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-facebook-like>
+  html_format: AMP
+  tag_name: "AMP-FACEBOOK-LIKE"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  satisfies: "amp-facebook-like"
+  requires: "amp-facebook-like extension .js script"
+  # data-* is generally allowed, but it's not generally mandatory.
+  attrs: {
+    name: "data-href"
+    mandatory: true
+  }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/social/amp-facebook-like"
```

Comment:
The spec_url is changing again and it's no longer necessary to specify a sub-component. So let's remove `social` from the URL. It will be this instead:
```
  spec_url: "https://www.ampproject.org/docs/reference/components/amp-facebook-like"
```